inginious/frontend/plugins/user_management/pages/api/user_status.py

```diff
@@ -0,0 +1,47 @@
+import web
+
+from inginious.frontend.plugins.user_management.collections_manager import CollectionsManagerSingleton
+from inginious.frontend.plugins.user_management.user_status import get_num_open_user_sessions, \
+    get_submissions_running, get_custom_test_running
+from inginious.frontend.plugins.utils import get_mandatory_parameter
+from inginious.frontend.plugins.utils.superadmin_utils import SuperadminAPI
+
+
+class UserStatusAPI(SuperadminAPI):
+    """ API to know the process that are running of a user """
+
+    def API_GET(self):
+        """ ger request. returns a list of the process that the user is running"""
+        self.check_superadmin_rights()
+        username = get_mandatory_parameter(web.input(), "username")
+        collection_manager = CollectionsManagerSingleton.get_instance()
+
+        user_status = {"username": username,
+                       "num_connections": get_num_open_user_sessions(username, collection_manager)}
+        submissions = get_submissions_running(username, collection_manager)
+        custom_test = get_custom_test_running(username, collection_manager)
+
+        self.config_processes_dict(submissions)
+        self.config_processes_dict(custom_test)
+
+        user_status["submissions"] = sorted(submissions, key=lambda k: (k["courseid"], k["taskid"]))
+        user_status["custom_test"] = sorted(custom_test, key=lambda k: (k["courseid"], k["taskid"]))
+
+        return 200, user_status
+
+    def config_processes_dict(self, processes):
+        """ This method apply a "format" to the process, get course name,
+        get the task name and apply a format for the date """
```

Comment:
```suggestion
        """ This method applies a "format" to the processes: it gets the course name
        and the task name to apply a format to the process date """
```

Can you also explain the purpose of this method, I'm not sure why this formatting is done.